Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,238 @@
+
+var randomIndex = Int.random(in: 0..<allTheWords.count)
+let randomWord = allTheWords[randomIndex]
+var hiddenWord: [Character] = Array(repeating: "_", count: randomWord.count)
+let alphabets: Set<Character> = Set("abcdefghijklmnopqrstuvwxyz")
+var guessMax = 6
+var indices: Set<Int> = []
+var lettersThatYouGuessed = [String]()
+var guesses = [Int]()
+
+var guessesIsCorrect = [Int]()
+
+
+
+print(randomWord)
+//var userContinue = readLine()?.lowercased()
+
+repeat{
+    
+let userEntered = readLine()?.lowercased() ?? "a" // readLine because user needs to enter characters - also make sure that it only accepts one character and that one character is part of the alphabet
+    
+    
+    if !randomWord.contains(userEntered) {
+        guessMax -= 1
+        print(guessMax)
+    }
+    
+//    print(guessMax)
+    
+// loop 1 - keeps track of the index of entered characters
+    for(index,char) in randomWord.enumerated() {
+    if Character(userEntered) == char {
+        indices.insert(index)
+        indices.count
+     
+//        print(lettersThatYouGuessed)
+    } else {
+        if Character(userEntered) != char {//...does not equal the character in the string
+        guesses.append(index)
+        guesses.count
+        print()//print what???
+    }
+}
+
+}
+
+    
+       lettersThatYouGuessed.append(userEntered)
+    print(lettersThatYouGuessed)
+// loop 2 - uses the position that was saved and adds it to the dashes
+for(index,_) in hiddenWord.enumerated() {
+    if indices.contains(index) {
+        hiddenWord[index] = Character(userEntered)
+        indices.remove(index)
+        print()//enter something here
+    } else {
+        
+    }
+}
+   
+print(String(hiddenWord)) // user sees a____
+
+
+// loop 3
+    if guessMax == 5 {
+        print("""
+            __________     \n
+            |        |
+        
+        """)
+    } else if guessMax == 4 {
+        print("|         |    \n")
+    } else if guessMax == 3 {
+        print("|         |    \n")
+    } else if guessMax == 2 {
+        print("|         0    \n")
+    } else if guessMax == 1 {
+        print("|        /|\\  \n")
+    } else if guessMax == 0 {
+        print("|        / \\   \n")
+    }
+    
+
+// 4
+if guessMax == 0 {
+    print("You lost")
+        print("Would you like to play again?")
+    
+    
+    if String(userEntered) == "yes" {
+        //then loop back to beginning
+        } else if String(userEntered) == "no" {
+        } // then end game (break?)
+    
+    
+        }
+
+    
+}while(guessMax>0)
+    
+  
+
+// make it so you can play it again after one try
+// After each guess the computer should print out how many guesses you have remaining. For full credit, make it actually print out a picture somewhat similar to the link above.
```

Comment:
I love that you have a checklist for yourself, make sure that you are checking off all of the requirements before submitting :) With that said, please remove any commented out or unused code before submitting.